packages/react-charting/src/components/TreeChart/TreeChart.base.tsx

```diff
@@ -477,7 +477,7 @@ export class TreeChartBase extends React.Component<ITreeProps, ITreeState> {
 
   constructor(props: ITreeProps) {
     super(props);
-    this._margin = { top: 30, right: 20, bottom: 30, left: 50 };
+    this._margin = { top: 30, right: 130, bottom: 30, left: 50 };
     this._width = this.props.width || 1500;
```

Comment:
Will it help if we add margin as a prop as we have in other charts.
Right margin of 130 might make it work on docsite but leave too much empty space in actual usage